tests/integration/pilot/gateway_conformance_test.go

```diff
@@ -90,6 +91,14 @@ func TestGatewayConformance(t *testing.T) {
 			}
 
 			features := suite.AllFeatures
+			if ctx.Settings().GatewayConformanceStandardOnly {
+				features = k8ssets.New[suite.SupportedFeature]().
```

Comment:
There is some maintenance risk here if new features are added. LGTM though

Reply:
Yep agreed; I can add a StableOnly set upstream